### PR TITLE
#286 Added CodeTriage badge to README

### LINKED ISSUE #286: Add CodeTriage badge to README
[Open Source Helpers](https://www.codetriage.com/techqueria/website)

```
[![Open Source Helpers](https://www.codetriage.com/techqueria/website/badges/users.svg)](https://www.codetriage.com/techqueria/website)
```

---

> ✅️ By submitting this issue, I have verified the following

- [x] Checked to [see if the issue has already been discussed before](https://github.com/techqueria/website/issues) 🤔️
- [x] Made sure enough details were provided if proposing a new feature 🔍️

### PR DESCRIPTION
**Pull request by sonali9696**
Closes #286 

This PR adds the CodeTriage badge to README
---

> ✅️ By submitting this PR, I have verified the following

- [x] Checked to [see if a similar PR has already been opened](https://github.com/techqueria/website/pulls) 🤔️
- [x] Reviewed the [contributing guidelines](https://github.com/techqueria/website/blob/master/.github/CONTRIBUTING.md) 🔍️
- [x] Added my name to the bottom of the list under the **Contributors** section in the [README.md](https://github.com/techqueria/website/blob/master/README.md) with a link to my personal website or GitHub profile 👥️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,8 @@
 
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Ftechqueria%2Fwebsite.svg?type=small)](https://app.fossa.io/projects/git%2Bgithub.com%2Ftechqueria%2Fwebsite?ref=badge_small)
 
+[![Open Source Helpers](https://www.codetriage.com/techqueria/website/badges/users.svg)](https://www.codetriage.com/techqueria/website)
```
Comment (fvcproductions) [inline README.md]: Can we have this added above FOSSA status instead of after? Looks good otherwise!

**Comment by sonali9696** [comment]
Hi @fvcproductions , it would be great if you could review the PR and let me of any changes to be done to proceed with it.

File: README.md (hunk 1 above)
Comment (sonali9696) [reply README.md]: Sure, will make the changes and push it

Comment (sonali9696) [reply README.md]: I have resolved the placement issue and pushed the commit to the same PR

**Comment by sonali9696** [comment]
The changes required to solve the placement issue have been done @fvcproductions

**Comment by fvcproductions** [comment]
Thanks for your changes! 👍

**Pull request merged**